K2Bridge.Tests.End2End/TestElasticClient.cs

```diff
@@ -134,12 +134,15 @@ public async Task<JToken> MSearch(string indexName, string jsonQueryFile, bool v
                 DeleteValue(result, "responses[*].hits.hits[*].highlight");
             }
 
-            // backend query isn't something we want to compare since it's unique to K2
+            // Backend query isn't something we want to compare since it's unique to K2
             DeleteValue(result, "responses[*]._backendQuery");
 
             // Ignore fields.hour_of_day (script_fields are currently no supported)
             DeleteValue(result, "responses[*].hits.hits[*].fields.hour_of_day");
 
+            // Normalize all float metric values with fixed number of decimal
+            NormalizeMetricValues(result, "responses[*].aggregations..buckets..value");
```

Comment:
sometimes bucket can have 'value' and sometimes 'values'

should we consider this as well?

Reply:
ok, so can you add a todo in the code? (With link to a new issue)

Reply:
Added workitem reference on source code.